Web Scarping/README.md

```diff
@@ -0,0 +1,34 @@
+# Web Scraping using Python
```

Comment:
All the modules should start with, what we are going to study, why we are going to study, and in the end, wrap up.